### PR TITLE
Feature/styled component integration

### PR DESCRIPTION
**Pull request by lmazurowski**

### CODE REVIEW

File: src/components/AppHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+export const AppHeader = styled.header`
+  background-color: #222;
```
Comment (przemyslawjanpietrzak) [inline src/components/AppHeader.tsx]: move to some common variables

File: src/components/AppHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  height: 150px;
+  padding: 20px;
+  color: white;
```
Comment (przemyslawjanpietrzak) [inline src/components/AppHeader.tsx]: as above

File: src/components/AppTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const AppTitle = styled.h1`
+  font-size: 1.5em;
```
Comment (przemyslawjanpietrzak) [inline src/components/AppTitle.tsx]: pls use rem instead of em

File: src/requests/components/RequestTableTBody.tsx [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import styled from 'styled-components';
+
+export const RequestTableTBody = styled.tbody`
```
Comment (przemyslawjanpietrzak) [inline src/requests/components/RequestTableTBody.tsx]: look like dead

File: src/requests/components/RequestTableTd.tsx [hunk 5]
```diff
@@ -0,0 +1,4 @@
+import styled from 'styled-components';
+
+export const RequestTableTd = styled.td`
```
Comment (przemyslawjanpietrzak) [inline src/requests/components/RequestTableTd.tsx]: looks like dead

File: src/requests/components/index.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { RequestTable } from './RequestTable';
+import { RequestTableTBody } from './RequestTableTBody';
+import { RequestTableTd } from './RequestTableTd';
+import { RequestTableTh } from './RequestTableTh';
+import { RequestTableTr } from './RequestTableTr';
+
+export {
```
Comment (przemyslawjanpietrzak) [inline src/requests/components/index.ts]: You can use `export * from 'file'` syntax (check starter repo)

File: src/status/components/StatusIndicator.tsx [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+import { State } from '../ServerStatus';
+
+export const StatusIndicator = styled.p<State>`
+  color: ${props => props.status === 'alive' ? 'green' : 'red'}
```
Comment (przemyslawjanpietrzak) [inline src/status/components/StatusIndicator.tsx]: type props argument

File: src/components/AppHeader.tsx (hunk 1 above)
Comment (lmazurowski) [reply src/components/AppHeader.tsx]: You mean as [theme](https://www.styled-components.com/docs/api#typescript) based variables?

Comment (przemyslawjanpietrzak) [reply src/components/AppHeader.tsx]: yes

File: src/requests/components/RequestTableTBody.tsx (hunk 4 above)
Comment (lmazurowski) [reply src/requests/components/RequestTableTBody.tsx]: it will be populated in another PR

File: src/requests/components/RequestTableTd.tsx (hunk 5 above)
Comment (lmazurowski) [reply src/requests/components/RequestTableTd.tsx]: it will be populated in another PR

File: src/theme/defaultTheme.ts [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import { DefaultTheme } from 'styled-components';
+
+export const defaultTheme: DefaultTheme = {
```
Comment (przemyslawjanpietrzak) [inline src/theme/defaultTheme.ts]: use `typeof defaultTheme` as type

**Pull request merged**